### DISCUSSION TITLE
[Question]:I run the file ”frigate-0.13.0\frigate\__main__.py“ using pycharm, but encountered the following error,But it already exists in the root directory /labelmap.txt

### DISCUSSION THREAD

**Discussion by depingchen** [blakeblackshear/frigate#12130] [Ask A Question]
### What is your question:

*************************************************************
*************************************************************
***    Your config file is not valid!                     ***
***    Please check the docs at                           ***
***    https://docs.frigate.video/configuration/index     ***
*************************************************************
*************************************************************
***    Config Validation Errors                           ***
*************************************************************
[Errno 2] No such file or directory: '/labelmap.txt'
Traceback (most recent call last):
  File "D:\modelfile\frigate-0.13.0\frigate\app.py", line 645, in start
    self.init_config()
  File "D:\modelfile\frigate-0.13.0\frigate\app.py", line 112, in init_config
    user_config = FrigateConfig.parse_file(config_file)
  File "D:\modelfile\frigate-0.13.0\frigate\config.py", line 1333, in parse_file
    return cls.parse_obj(config)
  File "pydantic\main.py", line 526, in pydantic.main.BaseModel.parse_obj
  File "pydantic\main.py", line 339, in pydantic.main.BaseModel.__init__
  File "pydantic\main.py", line 1066, in pydantic.main.validate_model
  File "pydantic\fields.py", line 439, in pydantic.fields.ModelField.get_default
  File "D:\modelfile\frigate-0.13.0\frigate\detectors\detector_config.py", line 74, in __init__
    **load_labels(config.get("labelmap_path", "/labelmap.txt")),
  File "D:\modelfile\frigate-0.13.0\frigate\util\builtin.py", line 147, in load_labels
    with open(path, "r", encoding=encoding) as f:
FileNotFoundError: [Errno 2] No such file or directory: '/labelmap.txt'

*************************************************************
***    End Config Validation Errors                       ***
*************************************************************

进程已结束,退出代码1

**Comment by depingchen** [comment]
If there are professionals willing to answer my questions, thank you very much

**Comment by NickM-27** [comment]
You can't run it inside of pycharm. Frigate needs to be run inside of a docker container, typically for development it's recommended to use vs code dev containers

**Reply by depingchen** [reply]
Thank you! I will try it.

**Reply by depingchen** [reply]
Excuse me, do I need to modify the  configuration file“docker-compose.ynl” or where do I need to modify it when building containers using VScode?

**Reply by depingchen** [reply]
Thank you, I have successfully started running it now.

**Comment by depingchen** [comment]
![19df76071f1c0ff6fbe7e2e193d4138](https://github.com/blakeblackshear/frigate/assets/147976628/ec520309-85c2-4a40-857b-95808fe0b9e5)
